src/UI/TextField.elm

```diff
@@ -378,13 +406,16 @@ newPassword onChange label currentValue =
         "Current password"
         model.value
 
+**NOTE**: Uses [`withPlaceholder "●●●●●●●●"`](#withPlaceholder) as default.
+
 -}
 currentPassword : (String -> msg) -> String -> String -> TextField msg
 currentPassword onChange label currentValue =
     input onChange
         label
         currentValue
         forCurrentPassword
+        |> withPlaceholder "●●●●●●●●"
```

Comment:
why force this? I didn't get it

Reply:
:sweat_smile:  It's the "best default", as it's what we'll use everywhere...